cdc/sink/mysql.go

```diff
@@ -716,6 +716,9 @@ func (s *mysqlSink) prepareDMLs(rows []*model.RowChangedEvent, replicaID uint64,
 
 func (s *mysqlSink) execDMLs(ctx context.Context, rows []*model.RowChangedEvent, replicaID uint64, bucket int) error {
 	failpoint.Inject("MySQLSinkExecDMLError", func() {
+		// Add a delay to ensure the sink worker with `MySQLSinkHangLongTime`
+		// failpoint injected is executed first.
+		time.Sleep(time.Second * 2)
```

Comment:
What about use the failpoint expr "1*2s -> return(true)"?

Reply:
`1*sleep(2000)->1*return(true)`
should be triggered two times, but the first time DMLs will be executed successfully